src/daemon/rpc_command_executor.cpp

```diff
@@ -2490,23 +2502,61 @@ static void append_printable_service_node_list_entry(cryptonote::network_type ne
     buffer.append(indent2);
     if (entry.last_uptime_proof == 0)
     {
-      buffer.append("Last Uptime Proof Received: Not Received Yet");
+      buffer.append("Last Uptime Proof Received: (Awaiting confirmation from network)");
     }
     else
     {
       buffer.append("Last Uptime Proof Received: ");
       buffer.append(get_human_time_ago(entry.last_uptime_proof, time(nullptr)));
+    }
 
-      buffer.append("\n");
-      buffer.append(indent2);
-      buffer.append("IP Address: ");
+    buffer.append("\n");
+    buffer.append(indent2);
+    buffer.append("IP Address & Port: ");
+    if (entry.public_ip == "0.0.0.0")
+    {
+        buffer.append("(Awaiting confirmation from network)");
+    }
+    else
+    {
```

Comment:
I like changing the IP to "(Awaiting confirmation from network)", but I think these storage port should remain separate because we'll also have lokinet port & quorumnet port before long.

Reply:
Actually never mind, this is fine on one line.  Later we can change it to something like:

1.2.3.4 :23456 (storage-server), :22025 (qnet), :22026 (lokinet)